revisions/epoch/fractional.go

```diff
@@ -28,6 +47,7 @@ func (EightHourly) GetData() Data {
 		"The last PR merge commit of eight-hour partition of the day, by UTC commit timestamp on master. E.g., epoch changes at 00:00:00, 00:08:00, etc..",
 		time.Hour * 8,
 		time.Hour * 8,
+		"DEPRECATED: The eight_hourly is being deprecated in favour of epochs that divide evenly by 24 hours.",
```

Comment:
This should say either "The eight_hourly epoch is being deprecated" or just "eight_hourly is being deprecated". I don't know which you'd prefer, so am not making the change myself.